stdpopsim/cli.py

```diff
@@ -329,6 +329,9 @@ def add_simulate_species_parser(parser, species):
     species_parser.add_argument(
         "-q", "--quiet", action='store_true',
         help="Do not write out citation information")
+    species_parser.add_argument(
+        "-d", "--dry-run", action='store_true', default=False,
```

Comment:
I think we want to keep ``-d`` for ``--demographic-model`` (see #309). Maybe "-D"?

Reply:
Sounds good